webapp/src/index.html

```diff
@@ -28,8 +28,8 @@
   <script type="text/javascript">
     //These will be injected at the server
     initialConfig = {
-//      serverURL: "http://malariagen.net/apps/ag1000g/phase1-AR3/api",
-      dataset: "Samples_and_Variants",
+      serverURL: window.location.protocol + "//"+window.location.host+window.location.pathname.replace(/\/[^\/]*$/,'/')+"api",
+      dataset: "Test1",
```

Comment:
Samples_and_Variants has become the defacto example/default dataset, and is bundled with the code, i.e.
https://github.com/cggh/panoptes/tree/master/sampledata/datasets/Samples_and_Variants